src/domain/home/Homepage.jsx

```diff
@@ -1,27 +1,47 @@
 import React, { useState, useEffect } from 'react';
-import { Intro, TheApp, ActiveCampaigns, HowItWorks } from './components/index';
-import classes from './Homepage.module.scss';
+import {
+    Intro,
+    TakeAction,
+    ActiveCampaigns,
+    WaveBreak,
+    HowItWorks,
+    Partners,
+    Newsletter,
+} from './components/index';
 
 const Homepage = () => {
-  const [campaigns, setCampaigns] = useState([]);
+    const [campaigns, setCampaigns] = useState([]);
+    const [partners, setPartners] = useState([]);
 
-  useEffect(() => {
-    const fetchCampaigns = async () => {
-      const resp = await fetch('https://api.now-u.com/api/v1/campaigns');
-      const campaigns = await resp.json();
-      setCampaigns(campaigns?.data);
-    };
-    fetchCampaigns();
-  }, []);
+    useEffect(() => {
+        const fetchCampaigns = async () => {
+            const resp = await fetch('https://api.now-u.com/api/v1/campaigns');
+            const campaigns = await resp.json();
+            setCampaigns(campaigns?.data);
+        };
+        fetchCampaigns();
 
-  return (
-    <div className={classes.homepageContainer}>
-      <Intro />
-      <HowItWorks />
-      <ActiveCampaigns campaigns={campaigns} />
-      <TheApp />
-    </div>
-  );
+        const fetchPartners = async () => {
+            const respPartners = await fetch(
+                'https://api.now-u.com/api/v1/organisations'
```

Comment:
Similar to `campaignsURL`, `organisationsURL` can be created.

Reply:
good catch